package.json

```diff
@@ -30,7 +30,8 @@
     "storybook": "NODE_OPTIONS=--max_old_space_size=6096 start-storybook --config-dir stories/config --port 6006",
     "build-storybook": "NODE_OPTIONS=--max_old_space_size=6096 build-storybook --config-dir stories/config",
     "preversion": "npm-run-all --parallel typecheck lint",
-    "version": "ALLOW_MISSING_VARIABLES=1 yarn build"
+    "version": "ALLOW_MISSING_VARIABLES=1 yarn build",
+    "prepare": "husky install"
```

Comment:
I don't think this is needed. It is required only for initial setup for the repo. Isn't it?